src/pages/mining.md

```diff
@@ -209,3 +209,49 @@ You can review the node logs with this command:
 ```bash
 docker logs -f stacks_miner
 ```
+
+## Optional: Running in Kubernetes with Helm
+
+In addition, you're also able to run a testnet node in a Kubernetes cluster using the [stacks-blockchain Helm chart](https://github.com/blockstack/stacks-blockchain/tree/master/helm/stacks-blockchain).
+
+Ensure you have the following prerequisites installed on your machine:
+
+- [Docker](https://docs.docker.com/get-docker/)
+- [minikube](https://minikube.sigs.k8s.io/docs/start/) (Only needed if standing up a local Kubernetes cluster)
+- [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/)
+- [helm](https://helm.sh/docs/intro/install/)
+
+### Generate keychain and get testnet tokens
+
+Generate a keychain:
+
+```bash
+docker run -i node:alpine npx blockstack-cli@1.1.0-beta.1 make_keychain -t
```

Comment:
is this the rust CLI? do we _need_ it here? all other docs use our `@stacks/cli` package for keygen. I'd love to avoid the confusion for devs if not absolutely needed

Reply:
The `blockstack-cli` is used in a couple other places on this page, I kept it the same for consistency. I can update it across the board to use `@stacks/cli`, but would prefer to make that change in a separate PR.

Reply:
Will update the commands to use `@stacks/cli` in my next commit